server/etcdserver/server_test.go

```diff
@@ -1540,6 +1544,127 @@ func TestPublishRetry(t *testing.T) {
 	<-ch
 }
 
+func TestPublishV3(t *testing.T) {
+	n := newNodeRecorder()
+	ch := make(chan interface{}, 1)
+	// simulate that request has gone through consensus
+	ch <- &applyResult{}
+	w := wait.NewWithResponse(ch)
+	ctx, cancel := context.WithCancel(context.TODO())
```

Comment:
It is convention only, but [docs](https://golang.org/pkg/context/#Background) suggests usage of `context.Background()` in tests